en/advanced_config/imu_factory_calibration.md

```diff
@@ -0,0 +1,30 @@
+# IMU Factory Calibration
+
+PX4 OEM manufacturers can perform an IMU factory calibration in order to store persistent values for the accelerometer, gyroscope and magnetometer biases into persistent EEPROM.
+This ensures that end users can always reset vehicle configurations and tuning to a safe state for flying.
+
+This procedure will write the following parameters to `/fs/mtd_caldata`: [CAL_ACC*](../advanced_config/parameter_reference.md#CAL_ACC0_ID), [CAL_MAG*](../advanced_config/parameter_reference.md#CAL_MAG0_ID), [CAL_GYRO*](../advanced_config/parameter_reference.md#CAL_GYRO0_ID).
+This data will then be used when the parameters are set (or reset) to their default values.
+
+:::warning
+This feature relies on the FMU having a dedicated EEPROM chip or an accompanying IMU PCBA that has sufficient space for the data.
+PX4 will store the data EEPROM at `/fs/mtd_caldata`, creating the file if necessary.
```

Comment:
The file exists either way. It's a good way to check if the board supports it if in doubt.

Reply:
@bkueng You're saying that if there is an EEPROM (or somewhere to store the file) this file will already be created?

Do you think it is necessary to go to the level of explaining how to check for board support? IF so, what's the best way - just run a file system exists() check?

Reply:
FYI, did https://github.com/PX4/PX4-user_guide/commit/5addb362935bc4902b720b3ac36d679d96e26064 to remove the "creating the file ..." bit.

Reply:
Yes I think it's a good check to be sure. You can do `ls /fs` on the shell, and the output should contain `mtd_caldata`

Reply:
Thanks very much. Added that in commit https://github.com/PX4/PX4-user_guide/commit/0d1571cab1cd6364a496e1b687d2f8fd47c2f2bc (meant to PR it, but oh well)